pkgs/by-name/th/the-dark-mod-unwrapped/package.nix

```diff
@@ -0,0 +1,95 @@
+{
+  lib,
+  stdenv,
+  fetchsvn,
+  cmake,
+  ninja,
+  xorg,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "the-dark-mod-unwrapped";
+  version = "2.12";
+
+  # We use the SVN repo as it contains built static binaries that are not
+  # present in the Git mirror. Currently Nixpkgs only supports a limited set
+  # of packages that can be statically linked (notably excluding ffmpeg, pulseaudio
+  # and pipewire), so we couldn't build everything from scratch even if we want to :(
```

Comment:
Not sure I understand this: can't we just make them `buildInputs` and/or link them into the expected place in the repo?

Reply:
No — it specifically expects *statically-linked* libraries (.a) which Nixpkgs has a very limited support for right now (PulseAudio, Pipewire and libav are all marked broken.) It is simply infeasible for me to fix static compilation for them here, but maybe if somehow they can be compiled then I'm open to devendoring them.